cli/build.rs

```diff
@@ -351,7 +351,7 @@ fn main() {
     panic!("Cross compiling with snapshot is not supported.");
   }
 
-  let symbols_path = std::path::Path::new(
+  let symbols_path = std::path::Path::new("napi").join(
```

Comment:
Last time we did that it broke the release/publish process.

Reply:
I think it was outside the CLI folder before?

Reply:
Oh, it was in napi_sym: https://github.com/denoland/deno/pull/16330/files#diff-f744e98f02b8c530a278d32cdb7dd6fed387c0ae06607944b61324d0af3b7697 -- I think this should be ok.

Reply:
Okay, let's try that